packages/example/.storybook/preview.js

```diff
@@ -8,14 +8,14 @@ export const parameters = {
 
 // add global var to control server rendering
 export const globalTypes = {
-  serverRendering: {
-    name: 'Server',
-    description: 'Server Rendering',
-    defaultValue: 'allowed',
+  renderMode: {
+    name: 'Render Mode',
+    description: 'Render template on the server or client',
```

Comment:
Shouldn't we include that this is rendering through "Twig" as now there is no reference on the build version that the "server" indicates Twig templates? You cannot tell this from looking at the build 🤷

Reply:
That's up to the project that implements it, this storybook doesn't care what types of templates are rendered on the server. In this repo I've done an `hbs` server, so mentioning `twig` here would technically even be incorrect :D

Reply:
Storybook doesn't run the server, it just connects to some existing server as part of your project, so it can support anything that you can dream of 😄